BHoM_UI/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MongoDB.Bson" version="2.4.4" targetFramework="net452" />
```

Comment:
Why is this necessary? Referring the `Serialization_Engine.dll` is not enough?

Reply:
I think that's been added by @adecler's changes.